docs/releases.md

```diff
@@ -16,6 +16,15 @@ Please follow individual releases for more information.
 
 Graphback 0.14 contains a lot of breaking changes that will improve and simplify your application code.
 
+#### CRUD Mutation return types are nullable
+
+Previously, the CRUD mutation resolvers did not allow returning null. These can now be null.
+
+```
```

Comment:
you can make it a "patch" so that it render nicely.

Reply:
Woah. Did not know about this :D